Modules/_ssl.c

```diff
@@ -645,11 +645,11 @@ PySSL_SetError(PySSLSocket *sslsock, int ret, const char *filename, int lineno)
         {
             if (e == 0) {
                 PySocketSockObject *s = GET_SOCKET(sslsock);
-                if (ret == 0 || (((PyObject *)s) == Py_None)) {
+                if (((PyObject *)s) == Py_None) {
```

Comment:
I think that the `ret` parameter should be removed. `PySSL_SetError()` is a static function, so we can do this.

Reply:
@serhiy-storchaka 
Yes, you are right. I was mistaken that Py_None would be the same as NULL. I will revise as your advice.